<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Send:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TCPserver">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TCPserver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_listen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_send">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receive">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_rece">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>receive:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SocketInfo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_socket">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Socket:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
   </widget>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
